pkg/template/registry/templateinstance/strategy.go

```diff
@@ -55,9 +55,19 @@ func (templateInstanceStrategy) PrepareForCreate(ctx apirequest.Context, obj run
 	templateInstance := obj.(*templateapi.TemplateInstance)
 
 	if templateInstance.Spec.Requester == nil {
-		user, _ := apirequest.UserFrom(ctx)
-		templateInstance.Spec.Requester = &templateapi.TemplateInstanceRequester{
-			Username: user.GetName(),
+		user, ok := apirequest.UserFrom(ctx)
+		if ok {
```

Comment:
Alas, in all the various threads here, I forgot the crucial state of affairs wrt all this ... until the upstream service catalog changes arrive where it actually authenticates and the user is properly set, we'll fail on the the actual instantiate as we'll come in with the wrong user.

That's where all the provision/bind request paramater map username setting came into play. 

Hence the TSB extended tests failed all over the place once I "ensure that if no user info is present, the fields are empty" in `PrepareForCreate`

I'll add another //TODO once upstream pull comment sections, put this code in.